tools/check/check_code_quality.sh

```diff
@@ -16,21 +16,6 @@
 # limitations under the License.
 #
 
-#######################################################################################################################
-# Check frozen class modification
-#######################################################################################################################
-
-echo "Check if frozen class modified"
```

Comment:
are we happy to rely on only danger for this check?

Reply:
since the previous check was not working correctly (see #6438), I think this is fine.
Once merged, I will rebase #6438 to check again that this is working.